source/configure-connection.txt

```diff
@@ -0,0 +1,179 @@
+.. _django-connection-configuration:
+
+==================================
+Configure Your Database Connection
+==================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect your Django project to MongoDB
+and configure the connection.
```

Comment:
FWIW this still reads "funny" to me … if you are connecting your Django project to MongoDB then you've already configured the connection. Feel free to override me because I can be NIT-happy.

Reply:
Reworded